### PR TITLE
Traduction de container-orchestration en FR

### LINKED ISSUE #1664: [FR] Localize container-orchestration for French
Localization of container-orchestration.md into french.

### PR DESCRIPTION
**Pull request by huats**
### Describe your changes
This PR localizes container-orchestration concept to French

### Related issue number or link (ex: `resolves #issue-number`)
Resolves #1664 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/fr/container-orchestration.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+## Ce que c'est
+
+L'orchestration de [Conteneurs](/container/) fait référence à la gestion et à l'automatisation du cycle de vie d'applications conteneurisées au sein d'environnements dynamiques.
+Réalisée à l'aide d'un orchestrateur de conteneurs (la plupart du temps, [Kubernetes](/kubernetes)), elle permet les déploiements, le passage à l'échelle (automatique), (l'auto-)remédiation et le monitoring.
+L'orchestration est une métaphore:
+L'outil d'orchestration dirige les conteneurs tel un chef d'orchestre, s'assurant que chaque conteneur (ou musicien) fait ce qu'il doit faire.
+
+## Problème auquel il répond
+
+Gérer des [microservices](/microservices), la sécurité, et la communication réseaux à grande échelle — et des [systèmes distribués](/distributed-systems) en général — est compliqué, pour ne pas dire impossible, manuellement.
```
Comment (fydrah) [inline content/fr/container-orchestration.md:16]: I would move "manuellement" just after "Gérer" --> "Gérer manuellement des microservices [...]", what do you think?

**Review by fydrah — CHANGES_REQUESTED** [review]

**Review by fydrah — APPROVED** [review]

File: content/fr/container-orchestration.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Problème auquel il répond
+
+Gérer manuellement des [microservices](/microservices), la sécurité, et la communication réseaux à grande échelle — et des [systèmes distribués](/distributed-systems) en général — est compliqué, pour ne pas dire impossible.
+L'orchestration de conteneurs permet aux utilisateurs d'automatiser toutes ces tâches de gestion.
+
+## Quelle en est l'utilité
+
+Les outils d'orchestration de conteneurs permettent aux utilisateurs de déterminer l'état d'un système.
+Tout d'abord, on déclare tel qu'il devrait être (i.e. x conteneurs, y pods, etc...).
```
Comment (Krast76) [inline content/fr/container-orchestration.md:22]: "on déclare comment il devrait être"

**Review by Krast76 — CHANGES_REQUESTED** [review]

File: content/fr/container-orchestration.md (hunk 2 above)
Comment (huats) [reply content/fr/container-orchestration.md:22]: Hum. Utiliser "comment" me semble plus du "parlé" au lieu de "tel" qui me semble plus écrit... Je suis pour garder "tel"

Comment (fydrah) [reply content/fr/container-orchestration.md:22]: Je suis de l'avis de Christophe sur ce coup là :)

**Review by Krast76 — APPROVED** [review]

**Pull request merged**